app/javascript/src/utilities/utils.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React from 'react'
+
+function CSRFToken ({win = window}: {win?: Window}) {
```

Comment:
you might want also to migrate the test from here spec/javascripts/Policies/components/CustomPolicy.spec.jsx

Reply:
done!